hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -639,3 +639,22 @@ tests:
             value: 103706112
           - labels: 'kubevirt_vmi_memory_used_bytes{container="virt-handler", name="vm-example-2", namespace="default", node="node-1"}'
             value: 234329980
+
+  # Excessive VMI Migrations in a period of time
+  - interval: 24h
```

Comment:
I think it should be 1h interval

Reply:
Yes, right.
I've changed the interval between the samples to 1 hour, and there are 13 samples of `0 1 ... 12` followed by 11 samples of 13. Totaling of 24 samples with an increase of more than 12 in 24 hours.